lms/djangoapps/mobile_api/social_facebook/courses/tests.py

```diff
@@ -0,0 +1,267 @@
+"""
+Tests for Courses
+"""
+
+from django.core.urlresolvers import reverse
+from rest_framework.test import APITestCase
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from courseware.tests.factories import UserFactory
+from xmodule.modulestore.tests.factories import CourseFactory
+from student.models import CourseEnrollment
+from student.views import login_oauth_token
+from social.apps.django_app.default.models import UserSocialAuth
+import httpretty
+import json
+from openedx.core.djangoapps.user_api.api.profile import preference_info, update_preferences
+
+
+class TestGroups(ModuleStoreTestCase, APITestCase):
+    """
+    Tests for /api/mobile/v0.5/courses/...
+    """
+
+    USERNAME_1 = "Daniel Eidan"
+    EMAIL_1 = "daniel@ebnotions.com"
+    PASSWORD_1 = "edx"
+    FB_ID_1 = "10155110991745300"
+
+    USERNAME_2 = "Marc Ashman"
+    EMAIL_2 = "marc@ebnotions.com"
```

Comment:
> The issue is that these need to be real FB id's so that the avatars can be loaded properly

Hmm... does that mean the unit tests are calling the FB servers to get the avatars?  Or am I misunderstanding your comment?